sql/2021/pwa/sw_adoption_over_time.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# SW adoption over time
+SELECT DISTINCT
+  SUBSTRING(yyyymmdd, 0, 4) || '-' || SUBSTRING(yyyymmdd, 5, 2) || '-' || RIGHT(yyyymmdd, 2) AS date,
```

Comment:
Here's a more standard way of formatting the date
```suggestion
  REGEXP_REPLACE(yyyymmdd, r'(\d{4})(\d{2})(\d{2})', r'\1-\2\-3') AS date,
```